src/grid.cpp

```diff
@@ -1456,9 +1459,9 @@ inline void limit_slope(real& ql, real q0, real& qr) {
 //    if (qr < q0 || q0 < ql) {
 	if (bool(qr < q0) != bool(q0 < ql)) {
 		qr = ql = q0;
-	} else if (tmp1 * (q0 - 0.5 * tmp2) > sqr(tmp1) / 6.0) {
+	} else if (tmp1 * (q0 - 0.5 * tmp2) > sqr(tmp1) * SIXTH) {
 		ql = 3.0 * q0 - 2.0 * qr;
-	} else if (-(sqr(tmp1) / 6.0) > tmp1 * (q0 - 0.5 * tmp2)) {
+	} else if (-(sqr(tmp1) * SIXTH) > tmp1 * (q0 - 0.5 * tmp2)) {
```

Comment:
Did you check if this needed? It's fine, but usually the compiler should catch this. If it's not, we should keep an eye out for this and make sure we lift constant divides wherever possible.